brave/brave-ios
# Fix #4049: Rename 'Brave Search beta' to 'Brave Search'.

## Summary of Changes


This pull request fixes #4049 

## Submitter Checklist:


## Test Plan:

1. Verify that it says `Brave Search` on search picker onboarding screen
2. Verify that it says `Brave Search` on search settings screen where you choose normal or private browisng SE

Upgrade path
1. Install 1.30.1 or any older build with Brave Search in it
2. Run the app, select Brave Search as default SE during onboarding or from search settings
3. Upgrade to 1.31
4. Verify that Brave Search is still set as the default SE

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
